snippets/serverless-about-kafka-broker.adoc

```diff
@@ -0,0 +1,24 @@
+// Text snippet included in the following modules and assemblies:
+//
+// * /modules/serverless-broker-types.adoc
+// * /serverless/develop/serverless-kafka-developer.adoc
+
+:_content-type: SNIPPET
+
+For production-ready Knative Eventing deployments, Red Hat recommends using the Knative Kafka broker implementation. The Kafka broker is an Apache Kafka native implementation of the Knative broker API that is backed by the `KafkaChannel` channel implementation.
+
+The Kafka broker supports any Kafka version, and has a native integration with Kafka for storing and routing events. This allows better integration with Kafka for the broker and trigger model over other broker types, and reduces network hops.
```

Comment:
In theory we do support some versions. Our upstream testing is done by Strimzi. 

The Strimzi currently used in Knative upstream supports Kafka 3.0 and later.
The new version of Strimzi already removed the support for Kafka 3.0, and has it just for 3.1 and 3.2